common/src/main/java/com/wynntils/screens/settings/WynntilsSettingsScreen.java

```diff
@@ -51,7 +51,6 @@ public class WynntilsSettingsScreen extends Screen {
 
     public WynntilsSettingsScreen() {
         super(new TranslatableComponent("screens.wynntils.settingsScreen.name"));
-        lastScreen = McUtils.mc().screen;
```

Comment:
I've said it before, but I liked the practice in Vanilla of always sending in a "parent" or "lastScreen" to a Screen constructor. If that is always used, it is easy to know where to return to, and the "chaining" of screens is very apparent.

That way, if we e.g. add a hotkey for going to settings, the Feature that implements it can call WynntilsSettingsScreen(null), and then we will exit right back with esc, which is the expected behavior in such a case, but if we got there from the wynntils main menu, we get back to it.

Reply:
Yeah, but this can also cause bugs (looping between 2 screens). I think it is safe to assume we don't open the settings screen from anywhere else for now.